data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,280 @@ event "remnant: pantica void sprites"
+				goto center
+			label centeralt
+			`	"Yes, that is correct." She signs firmly.`
+			label center
+			`	Her signs take on a rhythmic pace as if reciting something. "Sagittarius A* is a strong astronomical radio source and a supermassive black hole that is located at the center of the Milky Way. According to our best measurements, it has a mass of 3F6290 solar masses and is surrounded by a ring of gas and dust that can range from 63 to 24D2 degrees Kelvin."`
+			`	She returns to a conversational cadence. "Would you be interested in taking part in this expedition?"`
+			choice
+				`	"Sure, what do you need?"`
+					goto scienceequipment
+				`	"You want me to take you to a black hole?!"`
+				`	"Sorry, I'm not interested in visiting a black hole."`
+					decline
+			`	She looks momentarily surprised by your reaction. "Yes, exactly that. Current gravitational repulsor engines can nullify most of the gravitational pull, and an extra thermoelectric cooler or two should be sufficient to manage the heat. As such, the risks are mangeable so long as you do not try to approach the event horizon.`
+			label scienceequipment
+			`	"For this mission in specific, however, we will need some scientific equipment that is actually hardwired into your ship, not just carried along with us. When we arrive on <planet>, they will have the equipment ready to install. Just be aware that it will require six outfit space, and requires a scientist, so you will need a free bunk for them, too.`
+			`	"Once the laboratory module is installed, it will simply be a matter of traveling to Sagittarius A*. After our measurements, recordings, and observations are complete, we will return and report on our findings."`
+			`	She glances down at her half-finished meal. "I am already packed and ready to go. I will meet you onboard the <ship> as soon as I am done eating."`
+			branch dawnsymbiont
+				has "remnant symbionts info"
+			choice
+				`	"Okay, I will meet you onboard the ship."`
+					accept
+				`	"Developed quite the appetite, haven't you?"`
+			branch dawnlicense
+				has "license: Remnant Capital"
+			`	She shrugs. "I burn a lot of calories, and eating healthy is important. Besides, I don't know how good food will be on your ship." She resumes eating with vigor and does not seem inclined to continue the conversation.`
+				accept
+			label dawnlicense
+			`	"I am not yet used to the nutrient intake required to maintain my symbiont, so I have to consciously make sure that I eat enough to sustain it."`
+			choice
+				`	"Symbiont? What is that?"`
+					goto whatis
+				`	"What? You're joking, right?"`
+			`	"Suit yourself. I will meet you onboard once I'm done." She shrugs, and returns to eating."`
```

Comment:
I don't see how this follows from making the above choice... seems like two different conversations have been joined.

Reply:
This is referring to the "Suit yourself." response?

Basically, Dawn's not interested in convincing <first> of anything, nor does she feel the need to convince them of anything. If <first> is going to treat it like a joke or something unbelievable, then Dawn is just going to drop the subject.

Reply:
Yeah, I gathered that was the intent but it just sounds odd.

Reply:
I'm fine with this being left as is.